src/libertem/common/buffers.py

```diff
@@ -42,7 +42,7 @@ def empty_aligned(size, dtype):
 
 
 def zeros_aligned(size, dtype):
-    if dtype == np.object or np.prod(size, dtype=np.int64) == 0:
+    if dtype == object or np.prod(size, dtype=np.int64) == 0:
```

Comment:
Is `np.object` an alias of `object`? I.e. if a caller uses the old way, does this equality check hold?

Reply:
Yes it does. `np.object == object` holds. I tried that before changing. :-)

Reply:
Actually, even `np.object is object`. :-)